cmd/buildah/bud.go

```diff
@@ -77,35 +78,14 @@ func budCmd(c *cli.Context) error {
 			tags = tags[1:]
 		}
 	}
-	pull := true
-	if c.IsSet("pull") {
-		pull = c.BoolT("pull")
-	}
-	pullAlways := false
-	if c.IsSet("pull-always") {
-		pull = c.Bool("pull-always")
-	}
-	runtimeFlags := []string{}
-	if c.IsSet("runtime-flag") {
-		runtimeFlags = c.StringSlice("runtime-flag")
-	}
-	runtime := ""
-	if c.IsSet("runtime") {
-		runtime = c.String("runtime")
-	}
-
 	pullPolicy := imagebuildah.PullNever
-	if pull {
+	if c.BoolT("pull") {
 		pullPolicy = imagebuildah.PullIfMissing
 	}
-	if pullAlways {
+	if c.Bool("pull-always") {
```

Comment:
Are these options exclusive? If so, you should check for that. If not, could you add to the description of `pull-always` that it overrides `pull` if both are specified?

Reply:
No pull means pull if it is not there,  pull-always means pull even if it is there, I believe.

Reply:
👍

Reply:
Updated to say that it overrides pull.